internal/client/gitlab.go

```diff
@@ -48,6 +48,41 @@ func NewGitLab(ctx *context.Context, token string) (Client, error) {
 	return &gitlabClient{client: client}, nil
 }
 
+// CloseMilestone closes a given milestone.
+func (c *gitlabClient) CloseMilestone(ctx *context.Context, repo Repo, title string) error {
+	milestone, err := c.getMilestoneByTitle(repo, title)
+
+	if err != nil {
+		return err
+	}
+
+	if milestone == nil {
+		return fmt.Errorf("no open milestone matching: %s", title)
+	}
+
+	closeStateEvent := "close"
+
+	opts := &gitlab.UpdateMilestoneOptions{
+		Description: &milestone.Description,
+		DueDate:     milestone.DueDate,
+		StartDate:   milestone.StartDate,
+		StateEvent:  &closeStateEvent,
+		Title:       &milestone.Title,
+	}
+
+	_, _, err = c.client.Milestones.UpdateMilestone(
+		repo.String(),
+		milestone.ID,
+		opts,
+	)
+
+	if err != nil {
```

Comment:
`return err` :)

Reply:
Updated in https://github.com/goreleaser/goreleaser/pull/1657/commits/5aea6dd9fed3c81f9878bae771456b73722ba4a7 👍